### PR TITLE
[Tooling] Update release-toolkit to 3.0

### PR DESCRIPTION
**Pull request by AliSoftware**
As it says on the tin.

@jkmassel I expected to also have to update the CI config file (`.circleci/config.yml`) to install `drawText` via homebrew for the `promo-screenshots` job [here](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.circleci/config.yml#L282-L285) — as per [those instructions from the release-toolkit 3.0 update](https://github.com/wordpress-mobile/release-toolkit/pull/312/files#diff-e39c28e5e33301e7a94a866a62ffce85b4a42cadaa201837879de70cfeeaefd3R32), since that's the main breaking change that we need to adapt to when going thru that major version bump in clients.

But… I was unable to install it as brew tells me that there's no such formulae:
```
$ brew install automattic/build-tools/drawText
==> Tapping automattic/build-tools
Cloning into '/usr/local/Homebrew/Library/Taps/automattic/homebrew-build-tools'...
remote: Enumerating objects: 131, done.
remote: Counting objects: 100% (131/131), done.
remote: Compressing objects: 100% (129/129), done.
remote: Total 131 (delta 77), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (131/131), 5.54 MiB | 3.23 MiB/s, done.
Resolving deltas: 100% (77/77), done.
Tapped 3 formulae (15 files, 5.6MB).

Warning: No available formula or cask with the name "automattic/build-tools/drawtext". Did you mean automattic/build-tools/hostmgr?
```
And indeed I [can't find the formula in our brew tap](https://github.com/automattic/homebrew-build-tools) (nor a PR to add it)

What am I missing?

### CODE REVIEW

**Comment by mokagio** [comment]
For what is worth, there's also no formula in the main Homebrew. That's to be expected given the instructions say to add a custom formulae repository, but, since there's no formula there, I thought I'd check in the main one.

```
➜ brew search drawText
==> Formulae
libdrawtext
```

**Comment by AliSoftware** [comment]
@mokagio Yep, @jkmassel  confirmed to me in 1:1 that he forgot to push the formula yet, he'll look into that in the upcoming days to get it up in our own brew tap shortly.

Not sure if we should merge this PR right away (after all, you migrated to 3.0 on WPiOS already too) and make a subsequent PR in the future to make the CI install drawText for promo screenshots (which, by that time, might ultimately be on the Buildkite pipeline and scripts rather than in CircleCI like now if migration of promo screenshots job happens by then), or if we want to block the PR and update to the release toolkit until drawText lands… 🤷 

My 2 cents is, because WPiOS is already on 3.0 (and without drawText), that 3.0 is required to have the buildkite API trigger action that is needed to migrate to Buildkite anyway, and that it might make more sense to revisit the installation of drawText at the time Jeremy will migrate the promo-screenshot and all other jobs to Buildkite… I'd say let's merge this one PR and take note to solve the drawText install later, potentially after Buildkite migration lands. wdyt?

File: Gemfile.lock
Comment (mokagio) [inline Gemfile.lock:6]: Off topic: I'd never seen this kind of versioning 🤔 But Rails has been using it for quite some time, apparently:

**Review by mokagio — APPROVED** [review]

**Comment by mokagio** [comment]
>I'd say let's merge this one PR and take note to solve the drawText install later, potentially after Buildkite migration lands. wdyt?

This is what I think:

image

😄

**Comment by AliSoftware** [comment]
Ahaha 👍 
Ended up being a bad timing because Jeremy just DMed me that he finally [landed `drawText` in our brew tap](https://github.com/Automattic/homebrew-build-tools/commit/f7ea009f34ab0793398dc02f2bfdeaa78537ed71) overnight… eh, I'll just do a subsequent PR to land it at some point then 🙃

**Pull request merged**
